### ISSUE TITLE
Never get it to work

### ISSUE THREAD

**Issue by Chhoeachim** [flowtype/ide-flowtype#76]
I've tried many hours browsing through the internet. At last it still not work.
The lack of docs and description make it harder for newbie.

**Comment by mostafaeweda** [comment]
Are you using Atom >= 1.32.x ?
Could you have been impacted https://github.com/flowtype/ide-flowtype/issues/74?
That should've been fixed by https://github.com/flowtype/ide-flowtype/pull/75

**Comment by Chhoeachim** [comment]
I'm using Atom 1.32.1. Actually I've made it worked just now. I've opened this issue because I think the docs should provide more details about installation setups in order the package to work smoothly without much pains. 

What I was missing was just a small thing that I need to install `flow-typed` globally and run `flow-typed install` at the root of project and add `<PROJECT_ROOT>/flow-typed/`  under the `[libs]` in `.flowconfig`.

I hope this should improved just to make another much easier than this.
